### PR TITLE
PROC-1183 Add tracking information to shipment API docs

### PR DESCRIPTION
**Pull request by Smitsel**
Add the new tracking parameters to the docs.

### CODE REVIEW

File: source/reference/v2/shipments-api/create-shipment.rst [hunk 1]
```diff
@@ -41,6 +41,37 @@ Parameters
...
+
+       .. list-table::
+          :widths: auto
+
+          * - ``carrier``
+
+              .. type:: string
+                 :required: true
+
+            - Name of the postal carrier (as specific as possible). For example ``DHL US`` and not only ``DHL``.
```
Comment (willemstuursma) [inline source/reference/v2/shipments-api/create-shipment.rst]: Let's have a PostNL example here.

**Comment by Smitsel** [comment]
@ndijkstra can you check out the field descriptions if they are clear?

@sandervanhooft we've added some new parameters. So pinging you ;)

File: source/reference/v2/shipments-api/create-shipment.rst [hunk 2]
```diff
@@ -41,6 +41,37 @@ Parameters
 
             - The API resource token of the order line, for example: ``odl_jp31jz``
 
+   * - ``tracking``
+
+       .. type:: object
+          :required: false
+
+     - An object containing tracking details for the shipment. When sent the ``carrier`` and ``code`` parameters become required for this endpoint.
```
Comment (ndijkstra) [inline source/reference/v2/shipments-api/create-shipment.rst]: When sent, the ``carrier`` and ``code`` ...

**Comment by willemstuursma** [comment]
@Smitsel can you also add the `lines.quantity` parameter already.

File: source/reference/v2/shipments-api/get-shipment.rst [hunk 3]
```diff
@@ -68,6 +68,33 @@ Response
...
+
+              .. type:: string
+
+            - The track and trace code for the shipment.
+
+          * - ``url``
+
+              .. type:: string
+
+            - The URL the consumer can track the shipment.
```
Comment (ndijkstra) [inline source/reference/v2/shipments-api/get-shipment.rst]: The URL where your customer can track the shipment.

File: source/reference/v2/shipments-api/create-shipment.rst [hunk 4]
```diff
@@ -41,6 +41,37 @@ Parameters
...
+                 :required: true
+
+            - Track and trace code for the shipment. For example ``3SKABA000000000``
+
+          * - ``url``
+
+              .. type:: string
+                 :required: false
+
+            - The URL where the consumer can track the shipment, for example: ``http://postnl.nl/tracktrace/?B=3SKABA000000000&P=1016EE&D=NL&T=C``
```
Comment (ndijkstra) [inline source/reference/v2/shipments-api/create-shipment.rst]: The URL where your customer can track the shipment

File: source/reference/v2/shipments-api/create-shipment.rst [hunk 5]
```diff
@@ -41,6 +41,37 @@ Parameters
...
+                 :required: true
+
+            - Name of the postal carrier (as specific as possible). For example ``DHL US`` and not only ``DHL``.
+
+          * - ``code``
+
+              .. type:: string
+                 :required: true
+
+            - Track and trace code for the shipment. For example ``3SKABA000000000``
```
Comment (ndijkstra) [inline source/reference/v2/shipments-api/create-shipment.rst]: The track and trace code of the shipment.

**Comment by sandervanhooft** [comment]
> @sandervanhooft we've added some new parameters. So pinging you ;)
OMG 😉

**Comment by sandervanhooft** [comment]
> can you also add the lines.quantity parameter already.

@willemstuursma does this mean a single (physical) order line can be shipped in multiple shipments? Like I ordered 8 coffeecups, received first shipment 6 cups, received second shipment 2 cups?

**Comment by Smitsel** [comment]
@sandervanhooft yes, you can ship an order line in multiple shipments :)

**Comment by Smitsel** [comment]
I've updated the PR according to feedback @ndijkstra.
Also added the ``lines.quantity`` parameter.

**Pull request merged**